WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtilsWrapper.kt

```diff
@@ -0,0 +1,18 @@
+package org.wordpress.android.ui.uploads
+
+import org.wordpress.android.fluxc.model.SiteModel
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Injectable wrapper around UploadUtils.
+ *
+ * UploadUtils interface is consisted of static methods, which makes the client code difficult to test/mock.
+ * Main purpose of this wrapper is to make testing easier.
+ */
+@Singleton
```

Comment:
Any upside of using `Singleton` instead of `Reusable` here?

Reply:
Nope, copy-pasting mistake:). Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10356/commits/de673a9a9f11fcde78b35229da9f76fefb9cdddc

Reply:
Ahah, you taught me about `Reusable` I was really wondering what was going on there :)